soh/soh/GameMenuBar.cpp

```diff
@@ -658,6 +658,11 @@ namespace GameMenuBar {
                     ImGui::EndCombo();
                 }
 
+                UIWidgets::Spacer(0);
+
+                UIWidgets::EnhancementCheckbox("Allow multi-windows", "gEnableMultiViewports");
```

Comment:
Is it only the WiiU that doesn't support multi viewports? if that is the case, should I add a `#ifndef __WIIU__` around this toggle?

I'm not sure what rendered Switch uses and if/how multi view port works on it.

Reply:
I ended up adding the ifndef here since it seems only wiiu renderer does not support multi viewports as of now.

Reply:
We could expose something, but it would need to be different than `UseViewports()`, since that is changed to return this same CVar now.

Reply:
That wouldn't work right because for Windows, if a user turned the toggle off, then `UseViewports()` would then return false, which would cause the toggle to disappear and the user wouldn't be able to turn it back on.

I added and exposed a separate function `SupportsViewports()` that does not use the CVar to show/hide the toggle. and gets used inside `UseViewports()` now.